src/main/java/com/yeet/songr/models/Song.java

```diff
@@ -4,20 +4,26 @@
 
 @Entity
 public class Song {
+
     @Id
     @GeneratedValue(strategy= GenerationType.AUTO)
     private Long id;
 
     private String title;
-    private int length;
+
     private int trackNumber;
```

Comment:
Your `Song` model doesn't have an `artist` field. That's fine - it's not required in the assignment, and it's probably more efficient to just use the album's artist anyway. However, in your `songs.html`, you try to reference `song.artist`, which is causing an error because that field doesn't exist in your database. Whether you want to add an `artist` property or just have it populate from the song's `album` is up to you